### ISSUE TITLE
IMOS: LATITUDE and LONGITUDE in both 'var' and 'data_variable' checker output

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#48]
file 
SOOP/SOOP-SST/9V2768_Wakmatha/2015/IMOS_SOOP-SST_T_20150528T000000Z_9V2768_FV01_C-20150624T041657Z.nc

```
var                                    :3:   126/142 :
    LATITUDE                           :3:    16/18 :
        valid_max                      :3:     1/ 2 :
            check_attribute_type       :3:     0/ 1 : Attribute type is not same
                                                      as variable type
        valid_min                      :3:     1/ 2 :
            check_attribute_type       :3:     0/ 1 : Attribute type is not same
                                                      as variable type
.....

    data_variable                      :3:    11/16 :
        LATITUDE                       :3:     2/ 4 :
            valid_max                  :3:     0/ 1 :
                check_attribute_type   :3:     0/ 1 : Attribute type is not same
                                                      as variable type
            valid_min                  :3:     0/ 1 :
                check_attribute_type   :3:     0/ 1 : Attribute type is not same
                                                      as variable type

```

Is it normal to have both LATITUDE and LONGITUDE  variables in the **var** and **data_variable** group ?

**Comment by mhidas** [comment]
It's ok for the same variable to appear in multiple groups, but in this case it's exactly the same test being applied twice. This redundancy should be removed.

**Comment by mhidas** [comment]
Fixed in #58
